fake-kafka.gemspec

```diff
@@ -19,7 +19,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.required_ruby_version = '>= 2.7.0'
+
   spec.add_development_dependency "bundler", "~> 1.16"
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rubocop", "~> 1.45"
```

Comment:
In this case I just followed the original approach because I wanted to limit my changes as much as possible to adding CI with the basic checks, but please feel free to tackle those issues in a separate PR 😄